### PR TITLE
Features/redpandaupdate

### PR DESCRIPTION
**Pull request by entlein**
Updating redpanda chart to 5.8.8 and changing to using only diff values.yaml

### CODE REVIEW

File: Makefile_rke2 [hunk 1]
```diff
@@ -49,7 +49,7 @@ attack: copy-scripts create-bad exec
 redpanda: helm 
 	-$(HELM) repo add redpanda https://charts.redpanda.com
 	-$(HELM) repo update
-	-$(HELM) upgrade --install redpanda-src redpanda/redpanda -n redpanda --create-namespace --values redpanda/rke2values.yaml --set image.repository=docker.redpanda.com/redpandadata/redpanda-unstable --set image.tag=v24.1.1-rc4
+	-$(HELM) upgrade --install redpanda-src  redpanda-data/redpanda --version 5.8.8 -n redpanda --create-namespace --values redpanda/diffvalues.yaml
```
Comment (tobiasgrantner) [inline Makefile_rke2:52]: Is `redpanda/rke2values.yaml` also not needed anymore? Then we could delete it as well.

**Review by tobiasgrantner — APPROVED** [review]
Great 👍

File: Makefile_rke2 [hunk 2]
```diff
@@ -49,7 +49,7 @@ attack: copy-scripts create-bad exec
 redpanda: helm 
 	-$(HELM) repo add redpanda https://charts.redpanda.com
```
Comment (tobiasgrantner) [inline Makefile_rke2:50]: I just tried to deploy the changes on RKE and the repo needs to be renamed to `redpanda-data`.

Comment (entlein) [reply Makefile_rke2]: good catch, thanks

**Pull request merged**